python/ray/serve/scripts.py

```diff
@@ -323,6 +324,16 @@ def deploy(config_file_name: str, address: str):
         "as the ingress deployment."
     ),
 )
+@click.option(
+    "--reload",
```

Comment:
let's add a shorter alias for convenience, `"-r"`

Reply:
I have added the shorter alias.